ortools/sat/python/cp_model.py

```diff
@@ -956,31 +959,32 @@ def ObjectIsAFalseLiteral(literal):
     return False
 
 
-class CpModel(object):
+class CpModel:
     """Methods for building a CP model.
 
   Methods beginning with:
 
   * ```New``` create integer, boolean, or interval variables.
   * ```Add``` create new constraints and add them to the model.
   """
+    __model: cp_model_pb2.CpModelProto
 
     def __init__(self):
         self.__model = cp_model_pb2.CpModelProto()
         self.__constant_map = {}
 
     # Naming.
-    def Name(self):
+    def Name(self) -> str:
         """Returns the name of the model."""
         return self.__model.name
 
-    def SetName(self, name):
+    def SetName(self, name: str):
         """Sets the name of the model."""
         self.__model.name = name
 
     # Integer variable.
 
-    def NewIntVar(self, lb, ub, name):
+    def NewIntVar(self, lb, ub, name: str) -> IntVar:
```

Comment:
`def NewIntVar(self, lb: Union[float, IntVar], ub: Union[float, IntVar], name: str) -> IntVar:`

Reply:
I'll probably define it as:
```
IntegralT = Union[numbers.Integral, np.integer]
```
See:
https://github.com/google/or-tools/blob/8179abf8346b02c9aa39d356177fdbd6315f2c34/ortools/sat/python/cp_model_helper.py#L24-L26